src/goExtraInfo.ts

```diff
@@ -28,7 +28,12 @@ export class GoHoverProvider implements HoverProvider {
 			} else {
 				text = lines[0];
 			}
-			let hover = new Hover({ language: 'go', value: text });
+			let hoverTexts = [];
+			if (definitionInfo.doc != null) {
+				hoverTexts.push({ language: null, value: definitionInfo.doc});
```

Comment:
Sorry about that, didn't notice that there was a simple string constructor for this